### DISCUSSION TITLE
View and Report RSSI and Distance

### DISCUSSION THREAD

**Discussion by daniam75** [mKeRix/room-assistant#1079] [Q&A]
Is there way in the CLI to see an entities reported RSSI and distance?  Also, is there a way to get this information into home assistant?

**Comment by legovaer** [comment]
You can use the [RESTFUL sensor](https://www.home-assistant.io/integrations/sensor.rest/) and connect to [room assistant API](https://www.room-assistant.io/guide/api.html). 

Another solution can be using the MQTT sensor.
